src/app/screens/SignUp/styles.module.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../../scss/variables/colors';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  justify-content: center;
+}
```

Comment:
remember that you can use the global layout class, in this case "column center" in your element